manifests/database/postgresql.pp

```diff
@@ -10,6 +10,22 @@
     default => postgresql_password($::foreman::db_username, $::foreman::db_password),
   }
 
+  if $::foreman::db_root_cert {
+    $pg_cert_dir = "${::foreman::app_root}/.postgresql/"
+
+    file { $pg_cert_dir:
+      ensure => 'directory',
```

Comment:
You should manage owner and permission here as well. I've learned that it's better to be explicit about that than to rely on defaults. There could be a weird umask for example.